hack/update-vendor.sh

```diff
@@ -47,6 +47,15 @@ if [ -z "${BASH_XTRACEFD:-}" ]; then
   set -x
 fi
 
+# list_staging_repos outputs a sorted list of repos in staging/src/k8s.io
+# each entry will just be the $repo portion of staging/src/k8s.io/$repo/...
+function list_staging_repos() {
```

Comment:
Correctly listing these portably and sorted in one place had a nice side effect of simplifying the script a bit.
I implemented a couple variants of this, but this is probably the most robust version.

Reply:
moved to kube::util as I found the other vendor scripts need to do the same thing a lot..